### ISSUE TITLE
Issue about boxes nms

### ISSUE THREAD

**Issue by Markin-Wang** [luiszeni/Boosted-OICR#11]
Hi, I found a problem when runinig the code.
Error: 
`
return torch.ops.torchvision.nms(boxes, scores, iou_threshold)
RuntimeError: boxes should have 4 elements in dimension 1, got 5
`
This error is caused by this section of code in box_results_with_nms_and_limit function:
`
dets_j = np.hstack((boxes_j, scores_j[:, np.newaxis])).astype(np.float32, copy=False)
`
`
keep = nms(torch.tensor(dets_j), torch.tensor(scores_j), torch.tensor(cfg.TEST.NMS))
`
The first input of torchvision.ops.nms is predicted boxes with shape (N,4), but you stack the boxes and their scores and the shape of dets_j is (N,5).

**Comment by luiszeni** [comment]
Hi,

which version of pytorch and torchvision are you using?

**Comment by Markin-Wang** [comment]
> Hi,
> 
> which version of pytorch and torchvision are you using?

Thanks for your early reply.
My torch version is 1.6 and torchvision version is 0.7.0.
In addition, I replaced the 
`
nms(torch.tensor(dets_j), torch.tensor(scores_j), torch.tensor(cfg.TEST.NMS))
`
with
`
nms(torch.tensor(boxes_j.astype(np.float32, copy=False)), torch.tensor(scores_j), torch.tensor(cfg.TEST.NMS))
`
It works.
By the way, I also ran the code with torch version 1.2, no error occured, but I checked the inputs of torchvision.ops.nms function in torch 1.2 documentation and it is the same as version 1.6
I paste the links of nms function in version 1.6 and 1.2 here.
[NMS function for 1.6](https://pytorch.org/docs/stable/torchvision/ops.html)
[NMS function for 1.2](https://pytorch.org/docs/1.2.0/torchvision/ops.html)
This is very interesting and I guess that the implementations in torchvision version compatible with torch 1.6 and 1.2 are different.
